.travis.yml

```diff
@@ -7,3 +7,7 @@ addons:
   postgresql: "9.4"
 before_script:
   - mix ecto.create
+script:
+  - mix test
```

Comment:
Never mind I read the docs makes sense. If the script block isn't provided the default script block is `mix test`, and in the cases where I have seen them explicit call mix test was something like

```
script:
   - mix test
   - call something else
```

in that case I think you have to define it, because you are override the default script block.